tests/driver_my9221/main.c

```diff
@@ -45,16 +45,16 @@ int main(void)
     }
     /* run some tests */
     LOG_INFO("- light up all LEDs one by one.\n");
-    for (unsigned i=0; i < dev.params.leds; ++i) {
+    for (unsigned i=0; i < dev.params.leds; i++) {
```

Comment:
this change is unrelated

Reply:
you are right, late programming effects :)